### DISCUSSION TITLE
Google Search Results Not Updated After Username Change

### DISCUSSION THREAD

**Discussion by sarojkumar007** [community/community#18835] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi,

Recently I changed my username, which led me to some unwanted challenges. My new username is not found in google searches.
It shows the newer name but older links.

![image](https://user-images.githubusercontent.com/41164464/174423539-14deca5d-cef3-4d6c-ba93-14bd8d3cbef2.png)

Any Idea how to resolve this or is it automatically solve.

Let me know, how you guys deal with it, when you change usernames.

Thanks

**Comment by melvincwng** [comment, marked as answer]
Hi @sarojk18, you would have to wait till Google reindex/crawls your GitHub webpage again for the updated changes.

Perhaps you can check again on your end in the next couple of hours/days. 

It really depends on when Google reindexes or crawls your website.

Hope this clarifies.

**Reply by sarojkumar007** [reply]
Thanks. Let's hope for the best.

**Comment by sirgeon** [comment]
This will take some time to update, don't worry this isn't permanent.
